examples/cpp/CMakeLists.txt

```diff
@@ -44,7 +44,9 @@ EXAMPLE_CPP(ISSKeypoints              ${CMAKE_PROJECT_NAME})
 EXAMPLE_CPP(LineSet                   ${CMAKE_PROJECT_NAME})
 EXAMPLE_CPP(Log                       ${CMAKE_PROJECT_NAME})
 EXAMPLE_CPP(OdometryRGBD              ${CMAKE_PROJECT_NAME})
-EXAMPLE_CPP(Octree                    ${CMAKE_PROJECT_NAME})
+if (BUILD_GUI)
```

Comment:
BUILD_GUI should be OFF?

Reply:
should it not be off, when testing the offscreen rendering?
Thanks a lot!